third_party/move/move-model/src/options.rs

```diff
@@ -11,6 +12,9 @@ pub struct ModelBuilderOptions {
     /// part of the model.
```

Comment:
nit+: I get that this is not part of the PR - but this comment is confusing:
- What is considered "new compiler" - this will not be true forever.
- The variable name should include that info as well

Reply:
I removed mentioning of the compiler, its really what the variable says, that the move-model is used as the compiler. This flag will also go away once we retire the v1 compiler.